File: MainModule/src/main/java/com/qihuan/find/view/discover/DiscoverFragment.java
package com.qihuan.find.view.discover;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.qihuan.find.R;
import com.qihuan.commonmodule.base.BaseFragment;

/**
 * DiscoverFragment
 */
public class DiscoverFragment extends BaseFragment {

    public static DiscoverFragment newInstance() {
        return new DiscoverFragment();
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_news, container, false);
    }

}

File: module_discover/src/main/java/com/qihuan/discovermodule/view/DiscoverFragment.kt
package com.qihuan.discovermodule.view


import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.alibaba.android.arouter.facade.annotation.Route
import com.qihuan.commonmodule.base.BaseFragment
import com.qihuan.commonmodule.router.Routes
import com.qihuan.discovermodule.R

/**
 * DiscoverFragment
 *
 * @author Qi
 */
@Route(path = Routes.DISCOVER_FRAGMENT)
class DiscoverFragment : BaseFragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_discover, container, false)
    }

    companion object {

        fun newInstance(): DiscoverFragment {
            return DiscoverFragment()
        }
    }

}
